### PR TITLE
Update WebView2 SDK to 1.0.1264.42

### PR DESCRIPTION
**Pull request by krschau**
* Update the WebView2 SDK used from 1.0.1020.30 to 1.0.1264.42
* Update two API calls to new names
* WebView2 is no longer experimental
* Change the way we get SDK version to use Microsoft.Web.WebView2.Core.dll.
  * In the future, we should change getting the browser version to use the environment's static method, rather than the loader dll

### CODE REVIEW

File: dev/WebView2/InteractionTests/WebView2Tests.cs [hunk 1]
```diff
@@ -155,27 +139,28 @@ private static int GetInstalledBrowserVersion()
...
+                FileVersionInfo sdkVersionInfo = FileVersionInfo.GetVersionInfo(dllPath);
+                sdkBuildVersion = sdkVersionInfo.ProductBuildPart;
                 Log.Comment("WebView2Tests Init: Found {0}: version {1} [build version: {2}] (file: {3})",
-                            NativeMethods.LoaderName,
-                            loaderVersionInfo.ProductVersion,
-                            loaderBuildVersion,
-                            loaderVersionInfo.FileName
+                            dllName,
+                            sdkVersionInfo.ProductVersion,
+                            dllPath,
```
Comment (DmitriyKomin) [inline dev/WebView2/InteractionTests/WebView2Tests.cs:155]: > dllPath, [](http://example.com/codeflow?start=27&length=9)

Should be build version to match the string output

File: build/CopyFilesToStagingDir.ps1 [hunk 2]
```diff
@@ -38,6 +38,7 @@ PublishFile -IfExists $FullBuildOutput\Microsoft.UI.Xaml\Generic.xaml $FullPubli
 PublishFile -IfExists $FullBuildOutput\Microsoft.UI.Xaml.Design\Microsoft.UI.Xaml.Design.dll $FullPublishDir\Microsoft.UI.Xaml.Design\
 PublishFile -IfExists $BuildOutputDir\$Configuration\AnyCPU\Microsoft.UI.Xaml.FrameworkPackagePRI\Microsoft.UI.Xaml.pri $FullPublishDir\Microsoft.UI.Xaml.FrameworkPackagePRI\
 PublishFile -IfExists $BuildOutputDir\$Configuration\AnyCPU\MUXControls.Test.TAEF\*.dll $FullPublishDir\Test\
+PublishFile -IfExists $BuildOutputDir\$Configuration\$Platform\MUXControlsTestApp.TAEF\WebView2Loader.dll $FullPublishDir\Test\
```
Comment (DmitriyKomin) [inline build/CopyFilesToStagingDir.ps1:41]: Do we still need this?

**Review by DmitriyKomin — DISMISSED** [review]
:clock1:

File: dev/WebView2/InteractionTests/WebView2Tests.cs (hunk 1 above)
Comment (krschau) [reply dev/WebView2/InteractionTests/WebView2Tests.cs]: Updated

File: test/MUXControlsTestApp/TAEF/MUXControlsTestApp.TAEF.csproj [hunk 3]
```diff
@@ -19,4 +19,10 @@
   </ItemGroup>
   <Import Project="..\MUXControlsTestApp.Shared.projitems" />
   <Import Project="..\MUXControlsTestApp.Shared.targets" />
+  <Target Name="Binplace" AfterTargets="Build">
+    <ItemGroup>
+      <BinplaceItem Include="$(PkgMicrosoft_Web_WebView2)\runtimes\win-$(Platform)\native\WebView2Loader.dll" />
+    </ItemGroup>
+    <Copy SourceFiles="@(BinplaceItem)" DestinationFiles="@(BinplaceItem -> '$(OutDir)\%(Filename)%(Extension)')" />
+  </Target>
```
[the remark this answers is not in the document]
Comment (krschau) [reply test/MUXControlsTestApp/TAEF/MUXControlsTestApp.TAEF.csproj]: Apparently not. Removed.

File: build/CopyFilesToStagingDir.ps1 (hunk 2 above)
Comment (krschau) [reply build/CopyFilesToStagingDir.ps1:41]: Yes, although eventually we can clear out the other ones (let's not do that today).

**Pull request merged**